test/xpu_api/random/default_engine_params.h

```diff
@@ -0,0 +1,27 @@
+//===----------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------===//
+
+#ifndef _DEFAULT_ENGINE_PARAMS_H
+#define _DEFAULT_ENGINE_PARAMS_H
+
+// Default engine params
+namespace DefaultEngineParams
```

Comment:
As we can see, this resulted in massive changes and unnecessarily increased verbosity in all RNG tests.

I suggest to spin it off into a separate PR, or maybe even ask @paveldyakov and his team to propose a solution that they think is most appropriate (as they are the maintainers of these tests).